<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#labelTitleGPGSA, #labelTitleGPRMC {
font-size:15px;
font-weight: bold;
}

#labelStatement {
font-size:12px;
font-weight: bold;
color: red;
}

QMainWindow {
background-color: white;
}

QPushButton {
min-width: 54px;
border: 1px solid #DCDCDC;
text-align: center;
color: #444444;
font-size: 85%;
font-weight: bold;
height: 27px;
padding: 0 8px;
border-radius: 2px;
background-color: #F5F5F5;
background-image: -webkit-gradient(linear,left top,left
bottom,from(#f5f5f5),to(#f1f1f1));
background-image: -webkit-linear-gradient(top,#f5f5f5,#f1f1f1);
background-image: -moz-linear-gradient(top,#f5f5f5,#f1f1f1);
background-image: -ms-linear-gradient(top,#f5f5f5,#f1f1f1);
background-image: -o-linear-gradient(top,#f5f5f5,#f1f1f1);
background-image: linear-gradient(top,#f5f5f5,#f1f1f1);
text-decoration: none;
}

QPushButton:hover {
border: 1px solid #C6C6C6;
color: #333;
background-color: #F8F8F8;
background-image: -webkit-gradient(linear,left top,left bottom,from(#f8f8f8),to(#f1f1f1));
background-image: -webkit-linear-gradient(top,#f8f8f8,#f1f1f1);
background-image: -moz-linear-gradient(top,#f8f8f8,#f1f1f1);
background-image: -ms-linear-gradient(top,#f8f8f8,#f1f1f1);
background-image: -o-linear-gradient(top,#f8f8f8,#f1f1f1);
background-image: linear-gradient(top,#f8f8f8,#f1f1f1);
text-decoration: none;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelTextLatitude">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Latitude</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTextLongitude">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Longitude</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLatitude">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelLongitude">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonLauncher">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>97</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Launch</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonStop">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>97</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStatement">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRecord">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>97</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Record</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonStopRecord">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>97</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Rec</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTitleGPRMC">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>GPRMC</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTitleGPGSA">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>GPGSA</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLatitudeDir">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelLongitudeDir">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelGPGSA">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelTitlePDOP">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>PDOP</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTitleHDOP">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>HDOP</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPDOP">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelHDOP">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelTitleVDOP">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>VDOP</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelVDOP">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>120</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
